firebase_config.js

```diff
@@ -0,0 +1,28 @@
+// Import the functions you need from the SDKs you need
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import.meta.env;
```

Comment:
I don't think this is the correct use of this and maybe delete it here.

Reply:
fixed